java/offline-geocode/src/main/java/com/esri/arcgisruntime/sample/offlinegeocode/MainActivity.java

```diff
@@ -363,241 +325,90 @@ protected void onResume() {
 
   @Override
   protected void onDestroy() {
-    super.onDestroy();
     mMapView.dispose();
+    super.onDestroy();
   }
 
+  /**
+   * Define a listener to handle drag events.
+   */
   private class DragTouchListener extends DefaultMapViewOnTouchListener {
```

Comment:
Not sure if we are really using the DragTouchListener class. I believe this is to drag the pin to desired location? Its not working in this sample. hence the onSingleTapConfirmed function really doesn't do anything. Please correct me if I misunderstood here.

Reply:
ok got it, it works on single tap the pin and then tap hold and drag the pin.